src/Numerical/DiscontinuousGalerkin/LiftFlux.hpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function lift_flux.
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Variables.hpp"
+
+/// \ingroup DiscontinuousGalerkin
+/// \brief Lifts fluxes from an interface to a volume contribution.
+///
+/// This does a diagonal lift.  This is apparently an approximation.
```

Comment:
This does a diagonal lift, as we are choosing the integration quadrature points to be the same as the collocation points.  We do this as it is much faster, but it does decrease the accuracy of the method.  See Saul's paper.

Reply:
Here's a possible Doxy comment. I tried to say just enough to point the curious reader towards more detail, without actually getting involved in any technicalities. The ideal comment may have more (or less) detail than what I've written...

```
\brief Lifts the flux contribution from an interface to the volume.

The lifting operation takes the (d-1)-dimensional flux term at the
interface and computes the corresponding d-dimensional term in the
volume. SpECTRE implements an efficient DG method in which each
interface grid point contributes only to that same grid point of the
volume. 

\details
SpECTRE implements a DG method with a diagonalized mass matrix (also
known as a mass-lumping scheme). This choice gives a large
reduction in the computational cost of the lifting operation, however,
the scheme is slightly less accurate, especially when the grid is
deformed by non-trivial Jacobians. For more details on the
diagonalization of the mass matrix and its implications, see Saul's
DG formulation paper, especially Section 3:
https://arxiv.org/abs/1510.01190.

\note
(keep Will's note)
```